tools/istio-iptables/pkg/capture/run.go

```diff
@@ -279,9 +279,13 @@ func (cfg *IptablesConfigurator) Run() error {
 
 	defer func() {
```

Comment:
the istio-vm/istio-init iptables have this final dump of the iptables-save at info-level on master.
I'm tempted to move it to debug only but I'm retaining the same behavior for now.

Reply:
yeah that's more or less the only thing the init container does and it's not a log stream anyone looks at very often, so it's less of a concern there anyway.